spec/controllers/http_status_agent_spec.rb

```diff
@@ -7,7 +7,7 @@ class MockResponse < Struct.new(:status, :headers, :url)
 describe 'HttpStatusAgent' do
 
   let(:agent) do
-    Agents::HttpStatusAgent.new(:name => SecureRandom.uuid, :options => valid_params).tap do |a|
+    Agents::HttpStatusAgent.new( :name => SecureRandom.uuid, :options => valid_params).tap do |a|
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)